CMakeLists.txt

```diff
@@ -148,9 +181,11 @@ option(QUDA_MPI "build the MPI multi-GPU code" OFF)
 # ARPACK
 option(QUDA_ARPACK "build arpack interface" OFF)
 option(QUDA_ARPACK_LOGGING "enable ARPACK logging (not availible for NG)" OFF)
+option(QUDA_DOWNLOAD_ARPACK "Download ARPACK-NG software as requested by QUDA_ARPACK" OFF)
 
 # OpenBLAS
 option(QUDA_OPENBLAS "enable OpenBLAS" OFF)
+option(QUDA_DOWNLOAD_OPENBLAS "Download OpenBLAS software as requested by QUDA_OPENBLAS" OFF)
```

Comment:
This has been moved. @bjoo  can you please check that we don't have leftovers from merge conflicts in this file?
Just looking at the diff a few things should not be here.